Files: Restructure download options (#5578)
WeblateOrg/weblate: docs/user/files.rst

.. note::

    What options are available might be limited by :ref:`privileges`.

.. _download:

Downloading translations
------------------------

From the project or component dashboard, translatable files can be downloaded
in the :guilabel:`Files` menu.

> The first option is to download the original format as translated in the
> repository. In this case, any pending changes in the translation are committed
> and the up-to-date file is downloaded without conversion.

Reviewer: Some improvement:

```suggestion
The first option is to download the file in the original format as it is stored in the
repository. In this case, any pending changes in the translation are getting committed
and the up-to-date file is yield without any conversions.
```

Those are same as before, I believe all of them makes the snippet better, but I don't insist on any of those...

>     The available options might be limited by :ref:`privileges`.

Reviewer 2: ```suggestion
    Available options might be limited by :ref:`privileges`.
```

>     The available options might be limited by :ref:`privileges`.

Reviewer 2: ```suggestion
    Option availability might be limited by :ref:`privileges`.
```
Is there a way to actually solve this?
"Availability of options" is also a bit iffy.
It is possible to cut "the" here, but it is also possible to misread that as the options that are available are limited (like now).

.. _download:

Downloading translations
------------------------

From the project or component dashboard, translatable files can be downloaded
in the :guilabel:`Files` menu.

The first option is to download the original format as translated in the
repository. In this case, any pending changes in the translation are committed
and the up-to-date file is downloaded without conversion.

You can also download the translation converted into one of the widely used
localization formats. The converted files will be enriched with data provided
in Weblate; such as additional context, comments or flags. Several file formats

> are available via the :guilabel:`Files` ↓ :guilabel:`Customize download`:

Reviewer 2: ```suggestion
are available via the :guilabel:`Files` ↓ :guilabel:`Customize download` menu:
```

    The available options might be limited by :ref:`privileges`.

.. _download:

Downloading translations
------------------------

From the project or component dashboard, translatable files can be downloaded
in the :guilabel:`Files` menu.

> The first option is to download original format as translated in the

Reviewer 2: ```suggestion
The first option is to download the original format as translated in the
```
Like it better without "as it is"

> repository. In this case any pending changes in the translation are committed

Reviewer 2: ```suggestion
repository. Pending translation changes are committed
```
yes?

Reviewer 3: no

> and up-to-date file is download without any conversion.

Reviewer 2: ```suggestion
and the up-to-date file is downloaded without any conversion.
```

> and up-to-date file is download without any conversion.

Reviewer: Just some improvements:
```suggestion
The first option is to download original format as it is stored in the
repository. In this case any pending changes in the translation are getting committed
and the up-to-date file is yield without any conversions.
```

Reviewer 3: already changed in the meantime

Reviewer: should I repeat the suggestions, based on the current version?

Reviewer 2: I don't see why not.

> You can also download the translation converted into one of widely used

Reviewer 2: ```suggestion
You can also download the translation converted into one of the widely used
```

localization formats. The converted files will be enriched with data provided

> in Weblate such as additional context, comments or flags. Several file formats

Reviewer 2: ```suggestion
in Weblate; such as additional context, comments or flags. Several file formats
```
not sure

Reviewer 3: I like semicolons, it is possible to use here :)

are available via the :guilabel:`Customize download` in the :guilabel:`Files`

> menu:

Reviewer 3: Yes, thanks!

* gettext PO
* XLIFF with gettext extensions
* XLIFF 1.1

> * TermBase eXchange

Reviewer 2: The actual list says "TBX" and "TMX" (in nb_NO anyway)

>gettext PO
XLIFF med gettext-endelser
XLIFF 1.1
**TBX
TMX**
gettext MO
CSV
Excel Open XML
JSON
Android String Resource
iOS-strenger

Reviewer 3: It’s because nb_NO is a version behind this. You will see it for translations after the release
